CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+## Questions / Bugs / Feedback
+
+If you have questions about the library, found a bug or want to suggest a feature, please create an issue.
+
+## Documentation
+
+You can visit the [docs website](https://json-schema-form.vercel.app/), however its source is not in the repo yet, only the build (`/docs_build`). Our docs are still coupled to Remote's internal Design System and integration tests. The effort to decouple it at the moment is too high.
+
+So our strategy is to at each new PR, to update the docs internally and manually copy its build to this repo.
+
+## Setup
+
+- Install the dependencies:
+
+```bash
+npm install
+```
+
+- Run the tests:
+
+```
+npm test
+```
+
+## Development workflow
+
+### Creating a new branch
+
+Submit your branch pointing to `main`.
+
+Please, always add tests to your bug fixs and new features.
+
+### Testing the PR in your project
+
+It's possible to test your PR in your own projects, by publishing a `dev` release.
+
+Note that only core maintainers (Remoters) can publish new releases. If needed, ask us in the PR and we'll do it for you. Check #3 for the video walkthrough.
+
+1.  Locally run the script `npm run release:dev:patch` or `npm run release:dev:minor` depending on your changes.
+    a. You'll be shown what's the new version and prompt you if it's correct. Eg
+
+        ```
+        Creating a new dev...
+        :: Current version: 1.0.0
+        :::::: New version: 1.0.1-dev.20230516-175718
+        Ready to commit and publish it? (Y/n)
+
+        ```
+
+    b. Then it will ask for the `@remoteoss` OTP for the NPM access to publish it.
+    c. Done! 🎉
+
+Every `dev` release is [tagged as `dev`](https://docs.npmjs.com/cli/v9/commands/npm-publish#tag), which means it won't be automatically installed in your project by default.
+You must specify the exact version, for example:
+
+```bash
+npm install @remoteoss/json-schema-form@1.0.1-dev.20230516-175718
+```
+
+You can create as many dev releases as you need during the PRs, running the same command.
+
+### Merging a PR
+
+A PR needs all CI checks to pass.
+
+By default, prefer to "Squash and Merge" giving it a message that follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
+
+The final release is done after merge.
```

Comment:
Confirm for me:

It does not matter if a dev version gets into `main`, as long as it's followed up with the actual stable release. I don't need to necessarily revert any commits with the `package-lock.json`/`package.json` changes back to what the current stable version is.

Reply:
perfect 4 me